prody/proteins/interactions.py

```diff
@@ -1478,18 +1489,27 @@ def showInteractionsGraph(statistics, **kwargs):
     node_distance = kwargs.pop('node_distance', 5)
     font_size = kwargs.pop('font_size', 14)
     seed = kwargs.pop('seed', 42)
-    cutoff = kwargs.pop('cutoff', 3)
+    cutoff = kwargs.pop('cutoff', 0.1)
+    multiple_chains = kwargs.pop('cutoff', False)
```

Comment:
Should this really take from cutoff?

Reply:
I am using cutoff to remove those pairs of contacts which occurred only a few times. 0.1 means that it will display pairs that were detected in 10% frames of the trajectory (if that was a single contact between residues).

Reply:
But it shouldn't be what's used for setting the multiple chains kwarg

Reply:
Ohh.. you are right. I forgot to replace the name. Thanks.

Reply:
You're welcome

Reply:
Shall I fix it?

Reply:
Yes, please.

Reply:
done